packages/core/utils/src/prettyDiagnostic.js

```diff
@@ -0,0 +1,55 @@
+// @flow strict-local
+import type {Diagnostic} from '@parcel/diagnostic';
+
+import formatCodeFrame from '@parcel/codeframe';
+import mdAnsi from '@parcel/markdown-ansi';
+
+type AnsiDiagnosticResult = {|
+  message: string,
+  stack: string,
+  codeframe: string,
+  hints: Array<string>
+|};
+
+export default function prettyDiagnostic(
```

Comment:
Should this be in `@parcel/reporter-cli`? Is it used in any other places?

Reply:
It should probably be used by hmr and devserver to format the errors